src/main/include/utils/CommandUtils.h

```diff
@@ -333,14 +338,19 @@ static frc2::CommandPtr Intake2(IntakeSubsystem* intakeSubsystem,
                          intakeSubsystem->Out();
                          scoring->SpinVectorSide(ScoringDirection::PodiumSide);
                        }).ToPtr())
-              .AndThen(frc2::WaitCommand(0.08_s).ToPtr())
+              .AndThen(frc2::WaitCommand(0.05_s).ToPtr())
+              // .AndThen(frc2::InstantCommand([intakeSubsystem, scoring] {
+              //            intakeSubsystem->Stop();
+              //            scoring->Stop();
+              //          }).ToPtr())
+              // .AndThen(UnbiasNote(intakeSubsystem, scoring)
+              //              .RaceWith(
```

Comment:
Scary `RaceWith`. Will also terminate immediately since the `InstantCommand` only runs for a single loop.